trino-lb-core/src/config.rs

```diff
@@ -93,6 +93,12 @@ pub struct TrinoLbTracingConfig {
 
     #[serde(rename = "OTEL_EXPORTER_OTLP_PROTOCOL")]
     pub otlp_protocol: Option<opentelemetry_otlp::Protocol>,
+
+    /// TODO: Ideally [`opentelemetry_otlp::Compression`] would serialize and deserialize to
+    /// `gzip`. However, they currently de/ser to `Gzip`, which differs from [`opentelemetry_otlp::Compression::from_str`]
+    /// We should raise an upstream issue
+    #[serde(rename = "OTEL_EXPORTER_OTLP_COMPRESSION")]
+    pub otlp_compression: Option<opentelemetry_otlp::Compression>,
```

Comment:
I think I brought this up before, but these settings are already handled by the OpenTelemetry SDK and OTLP crates. So this is double-handling.

Not a blocker, just wanted to mention it again.

Reply:
I should mention, this will be incompatible with stackable-telemetry (which leaves those settings to be applied via env vars) if you end up using that.

Reply:
Yes, I'm aware of that.
trino-lb supports both variants, I personally it being part of the application config. Users can choose :)